docs/GettingStarted.md

```diff
@@ -199,9 +199,13 @@ This section shows how to use eBPF for Windows in a demo that lets us control a
 
 #### Prep
 
-1. Build the ``port_leak`` and ``port_quota`` applications from under the tools project.
-1. Copy both the exe's to a machine that has eBPF installed. See
-   [Installing eBPF for Windows](#installing-ebpf-for-windows)
+1. Build the solution with the Release configuration. (Debug will also work for the usermode components.)
+1. Copy the entire contents of the build folder to your test machine; this is more than is minimally necessary, but
```

Comment:
i think we have a deploy script that does this already, should be linked in the docs, if it's not we should change that

Reply:
The deploy scripts that I was able to find all relied on running a local hyper-v hosted VM; I'm testing with a VM on a remote server I connect to over TShell, so that wasn't available to me. Either way, Method 1 doesn't set up the usermode service, which... wasn't clear to me at all initially, and definitely caused me some delays.

Reply:
I did not see any such checkbox when installing from my local Release build; I made sure not to build it as the Native-Only option, turned on all the options in the installer, and the service still isn't present. I'm pretty sure it's not even included in the MSI at all.

Reply:
Please file an issue

Reply:
Opened https://github.com/microsoft/ebpf-for-windows/issues/3215 -- I didn't have a chance to get to that yesterday.